src/modules/cmdpal/exts/Microsoft.CmdPal.Ext.WebSearch/Helpers/DefaultBrowserInfo.cs

```diff
@@ -0,0 +1,207 @@
+// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Text;
+using System.Threading;
+
+namespace Microsoft.CmdPal.Ext.WebSearch.Helpers;
+
+/// <summary>
+/// Contains information (e.g. path to executable, name...) about the default browser.
+/// </summary>
+public static class DefaultBrowserInfo
```

Comment:
this was a global helper for supporting reuse in other plugins. why making it exclusive?